src/store/Transaction.ts

```diff
@@ -145,8 +199,20 @@ export default {
         ) => {
             state.partialTransactions = conditionalSort(transactions, transactionComparator);
         },
-        setDisplayedTransactionStatus: (state: TransactionState, displayedTransactionStatus: TransactionGroupState) => {
-            state.displayedTransactionStatus = displayedTransactionStatus;
+        setAllTransactions: (state: TransactionState) => {
+            state.transactions = [...state.confirmedTransactions, ...state.unconfirmedTransactions, ...state.partialTransactions];
```

Comment:
I think the correct order would be => partial - unconfirmed - confirmed (Since in the tx list, partial and unconfirmed txs need to always go first.)

Reply:
done